gulpfile.js

```diff
@@ -9,48 +9,36 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 */
 
 const gulp = require('gulp');
+const rename = require('gulp-rename');
+const replace = require('gulp-replace');
 const del = require('del');
-const { exec } = require('child_process');
-const { renameSync: move } = require('fs');
-const renamer = require('renamer');
-const replace = require('replace');
+
+/**
+ * Cleans the PRPL server build directory (<project folder>/server/build)
+ */
+gulp.task('prpl-server:clean', () => {
+  return del('server/build');
+});
 
 /**
  * Builds the PRPL-server-ready version of the PWA, auto setting the base path
  * and renaming the node_modules folder, otherwise services like App Engine won't
  * upload it
  */
```

Comment:
We should update the comment since the gulp task no longer calling “polymer build —auto-base-path”. It’s just post build stuffs like moving and renaming.